src/os/bluestore/BlueStore.cc

```diff
@@ -7170,39 +7110,26 @@ void BlueStore::_txc_state_proc(TransContext *txc)
     case TransContext::STATE_KV_SUBMITTED:
       txc->log_state_latency(logger, l_bluestore_state_kv_committing_lat);
       txc->state = TransContext::STATE_KV_DONE;
-      _txc_finish_kv(txc);
+      _txc_committed_kv(txc);
```

Comment:
Okay, I forgot the onreadable callback's role.
I thought the old code where o->flush() blocks read(), because onreadable callback happened before data is applied to storage. Now, with temporary buffer cache, o->flush() isn't responsible for durability anymore.

Thank you for detail description.